docs/BR.md

```diff
@@ -2800,12 +2800,6 @@ Before including such an attribute, the CA SHALL:
   * Document the attributes within Section 7.1.4 of their CP/CPS, along with the applicable validation practices.
   * Ensure that the contents contain information that has been verified by the CA, independent of the Applicant.
 
-#### 7.1.4.2 Subject Information - Subscriber Certificates
```

Comment:
Note: there is a jump in the numbering from 7.1.2.9.5 to 7.1.4.2

Reply:
Thanks for the reminder! While I removed the 7.1.4.2, I've held off renumbering 7.1.2.4 (to 7.1.2.3) and 7.1.2.5 (to 7.1.2.4) while I wait for feedback from folks, since we make reference to those in a number of other places outside of Section 7.1, and so wanted to minimize the diff churn.

But yeah, the numbering will need to be fixed up.